docker/puppetdb/spec/puppetdb_spec.rb

```diff
@@ -106,6 +107,11 @@ def start_puppetdb
 
   before(:all) do
     @mapped_ports = {}
+    # Windows doesn't have the default 'bridge network driver
+    network_opt = File::ALT_SEPARATOR.nil? ? '' : '--driver=nat'
+
+    @network = %x(docker network create #{network_opt} puppetdb_test_network).chomp
+
     @postgres_container = run_postgres_container
 
     @pdb_image = ENV['PUPPET_TEST_DOCKER_IMAGE']
```

Comment:
Not something you introduced, but...

Can we move this up to be the first part of `before(:all)` ?

We should fail-fast if the image name isn't provided, before we start creating anything or running containers (or doing anything, really).

Reply:
updated in edffe1d13c04f56dda2de3922c08780545020f3d